plugin/src/main/java/org/opensearch/ml/sdkclient/SdkClientModule.java

```diff
@@ -80,7 +80,8 @@ protected void configure() {
                 return;
             case AWS_DYNAMO_DB:
                 log.info("Using dynamo DB as metadata store");
-                bind(SdkClient.class).toInstance(new DDBOpenSearchClient(createDynamoDbClient()));
+                bind(SdkClient.class)
+                    .toInstance(new DDBOpenSearchClient(createDynamoDbClient(), new RemoteClusterIndicesClient(createOpenSearchClient())));
```

Comment:
When you create a DDB client, do you see any sdk credential loading failures due to security issues? Last year when we tried to use aws sdk, the credential chain provider needs to read a file on the disk that is restricted access by OS so the SDK cannot be initialized. Wondering if this is a bug already fixed this year,

Reply:
@Zhangxunmt I was struggling with exactly that when trying to configure this client.

I was able to do a POC using AWS creds [here](https://github.com/opensearch-project/flow-framework/pull/631) and there were quite a few settings I had to add to [plugin-security.policy](https://github.com/opensearch-project/flow-framework/pull/631/files#diff-214b5d6612e51e2474aafa928918b67b571bfa31d6f4e398248840e5437714e9) and even that I needed to hard code a root path that I had permission to.